specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-06-01/Accounts.json

```diff
@@ -187,6 +187,9 @@
           "items": {
             "$ref": "#/definitions/MetricDimension"
           },
+          "x-ms-identifiers": [
+            "name"
+          ],
           "description": "The metric dimensions.",
           "readOnly": true
```

Comment:
Same here MediaServiceProperties - > storageAccounts is also an array. does this not need x-ms-idenitfiers. Can you explain a little on which scenario should we be adding this new property?

Reply:
The type object of storageAccounts, which is sotrageAccount in this case, has a property named "id" already, which is the default value expected when x-ms-identifiers is not specified. 

Same reason for AccountFilterCollection.

Hope this helps :)